CHANGELOG.md

```diff
@@ -3,6 +3,80 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.17.0
+
+_Sep 1, 2022_
+
+🎉 We are excited to finally introduce a stable release (v5.0.0) for the `@mui/x-date-pickers` and  `@mui/x-date-pickers-pro` packages!
+
+If you are still using picker components from the `lab`, take a look at the [migration guide](https://mui.com/x/react-date-pickers/migration-lab/).
+
+We'd like to offer a big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Implement Lazy loading (#5214) @DanailH
+
+  Pro users now can try experimental `lazyLoading` feature.
+  Implement lazy infinite skeleton loading with a few settings.
+
+  To enable this feature, add `experimentalFeatures={{ lazyLoading: true }}`.
+  Lazy loading feature has a few other prerequisites.
+  See the [documentation](https://mui.com/x/react-data-grid/row-updates/#lazy-loading) to explore example in detail.
+
+- 🚀 Improve `pickers` focus management (#5820) @alexfauquette
+- 🎉 Enable disabling `day` on date range picker depending on `position` (#5773) @alexfauquette
+- ✨ Various `DataGrid` and `pickers` improvements
```

Comment:
"pickers" is not a component.

```suggestion
- ✨ Various `DataGrid` and pickers improvements
```

Reply:
Since we are going to have this sentence in future releases, I propose to talk about the concept and not the components here

```suggestion
- ✨ Various improvements on the grid and on the date and time pickers components
```

or simply

```suggestion
- ✨ Various improvements
```